primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js

```diff
@@ -842,18 +842,20 @@
         },
 
         /**
-         *  Scrolls to a component with given client id
+         * Scrolls to a component with given client id
          * @param {string} id The ID of an element to scroll to.
+         * @param {string | number | undefined} duration string or number determining how long the animation will run. Default to 1000
          */
-        scrollTo: function(id) {
+        scrollTo: function(id, duration) {
             var offset = $(PrimeFaces.escapeClientId(id)).offset();
             var scrollBehavior = 'scroll-behavior';
             var target = $('html,body');
             var sbValue = target.css(scrollBehavior);
+            var animationDuration = duration || 1000;
```

Comment:
```suggestion
            var animationDuration = duration || 400;
```
https://github.com/primefaces/primefaces/issues/10275#issuecomment-1614802000